composer.json

```diff
@@ -17,11 +17,25 @@
         "psr-4": { "Omnipay\\Windcave\\" : "src/" }
     },
     "require": {
-        "omnipay/common": "^3"
+        "php": "^8.1",
```

Comment:
Same question as in https://github.com/digistorm/omnipay-westpac-payway-rest/pull/1
Should this be 8.2? Or maybe its compatible with 8.1 as well?

Reply:
added a comment here https://github.com/digistorm/omnipay-westpac-payway-rest/pull/1